ducttape/data_sources/lexia.py

```diff
@@ -74,6 +74,7 @@ def _login(self):
         elem.clear()
         elem.send_keys(self.username)
         elem.send_keys(Keys.RETURN)
+        time.sleep(2)
```

Comment:
Is it definitely not possible to do this with a `WebDriverWait()`?

Reply:
I didn't try, but I'm not sure what to wait for - the next line is already waiting for the password box. When I ran the code in Jupyter, I can watch it fill the username box, then I get a white screen flicker, then the next screen that's visible is line 83, where it reloads the username page.  The if clause in line 85 doesn't seem to catch that, and I end up timing out in `download_url_report` where it's looking for a button that's not there.

Reply:
Hmm, that's weird. Have you tried with a newer version of chrome/chromedriver/selenium?

Reply:
Selenium 3.141.0, updated Chrome to Chrome 83.0.4103.106, got the most recent Chrome 83 chromedriver (83.0.4103.39), still the same result.  In about 10 tests, 1 of them managed to get the password screen and complete.

Reply:
cool, it's probably not worth spending any more time on this right now. I'll approve and merge.